ChocolateyForBusinessQuickStartGuide.md

```diff
@@ -196,33 +217,37 @@ Run this script in a PowerShell Administrator console, to create the license pac
     ```powershell
     Set-ExecutionPolicy Bypass -Scope Process -Force; iex ((New-Object System.Net.WebClient).DownloadString('http://<_FQDN_OF_REPOSITORY_>:8081/repository/choco-install/Install.ps1'))
     ```
+
 1. Add the Nexus repository as a Chocolatey source:
 
     ```powershell
-    choco source add --name="'choco-hosted'" --source="'http://<_IP_OF_REPOSITORY_>:8081/repository/choco-hosted/'"
+    choco source add --name="'choco-hosted'" --source="'http://<_FQDN_OF_REPOSITORY_>:8081/repository/choco-hosted/'"
     ```
+
 1. Remove Chocolatey Community Repository as a souce:
 
     ```powershell
     choco source remove --name="'chocolatey'"
     ```
+
 1. Install the `chocolatey-license` package:
 
     ```powershell
-    choco install chocolatey-license
+    choco install chocolatey-license -y
     ```
+
 1. Install the `chocolatey.extension` package:
 
     ```powershell
-    choco install chocolatey.extension
+    choco install chocolatey.extension -y
     ```
 
 ### Central Management Server
 
-Again, the setup of the Chocolatey Central Management server is beyond the scope of this document (th. Please read over and follow the detailed instructions [[here|FeaturesChocolateyCentralManagement]]).
+Again, the setup of the Chocolatey Central Management server is beyond the scope of this document (please read over and follow the detailed instructions on the [Chocolatey Central Management Setup page](https://chocolatey.org/docs/central-management-setup).

```

Comment:
either works - use a full link like this ONLY if you plan to use this guide somewhere else. If it only exists here, use the link in the way I provided, but change it to `[[something something|CentralManagementSetup]]`.